### PR TITLE
Unknown port types handled with `unknown` instead of `Present`

### PR DESCRIPTION
**Pull request by lhstrh**
The type constraint `extends Present` (with `Present` being the type that included all types but `undefined`) was inconvenient and non-standard from the perspective of TypeScript. Removing this awkwardness will probably also make it more straightforward to offer support for generic reactors in LF. See https://github.com/lf-lang/reactor-ts/pull/168 for related changes made in the runtime.

### CODE REVIEW

File: .github/workflows/ts-tests.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
         if: ${{ runner.os == 'macOS' || runner.os == 'Linux' }}
       - name: Perform TypeScript tests
         run: |
-          ./gradlew targetTest -Ptarget=TypeScript -Druntime="git://github.com/lf-lang/reactor-ts.git#master"
+          ./gradlew targetTest -Ptarget=TypeScript -Druntime="git://github.com/lf-lang/reactor-ts.git#axmmisaka/remove-present-replace-unknown"
```
Comment (lhstrh) [inline .github/workflows/ts-tests.yml:40]:
```suggestion
          ./gradlew targetTest -Ptarget=TypeScript -Druntime="git://github.com/lf-lang/reactor-ts.git#master"
```

File: core/src/main/resources/lib/ts/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "LinguaFrancaDefault",
     "type": "commonjs",
     "dependencies": {
-        "@lf-lang/reactor-ts": "git://github.com/lf-lang/reactor-ts.git#master",
+        "@lf-lang/reactor-ts": "git://github.com/lf-lang/reactor-ts.git#axmmisaka/remove-present-replace-unknown",
```
Comment (lhstrh) [inline core/src/main/resources/lib/ts/package.json:5]:
```suggestion
        "@lf-lang/reactor-ts": "git://github.com/lf-lang/reactor-ts.git#master",
```

**Review by lhstrh — COMMENTED** [review]
Looks good, modulo suggested changes (to be applied after merging https://github.com/lf-lang/reactor-ts/pull/168)

**Review by axmmisaka — APPROVED** [review]
lgtm, especially considering it works :^)

**Pull request merged**